angular/projects/spark-angular/src/lib/components/sprk-dropdown/sprk-dropdown.component.ts

```diff
@@ -62,9 +63,14 @@ import { ISprkDropdownChoice } from './sprk-dropdown.interfaces';
             >
             <sprk-icon
               [iconType]="triggerIconType"
-              additionalClasses="sprk-c-Icon--stroke-current-color sprk-u-mls sprk-c-Icon--toggle sprk-Stack__item {{
-                additionalIconClasses
-              }}"
+              additionalClasses="
```

Comment:
I'm curious about the difference of when we use a function like `getAdditionalIconClasses` vs using this way.
It seems like they accomplish similar things, and I wonder if we should pick one for consistency.

Also, doesn't HAVE to be fixed, but perhaps a later issue

Reply:
I definitely like the idea of looking at it and figuring out the best way. I know that I have used `getClasses` when there is logic involved with what classes are being applied. But agree, separate issue related to API updates.